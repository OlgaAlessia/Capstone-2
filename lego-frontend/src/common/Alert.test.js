import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";

it("renders without crashing", function() {
  render(<Alert />);
});

it("matches snapshot works", function() {
  let messages = ["Everything is awesome!"];
  const { asFragment } = render(<Alert type="success" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot with an error ", function() {
  let messages = ["Houston we have a problem !"];
  const { asFragment } = render(<Alert type="danger" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});


