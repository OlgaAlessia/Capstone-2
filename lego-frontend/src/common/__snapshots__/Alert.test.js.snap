// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with an error  1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Houston we have a problem !
    </p>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot works 1`] = `
<DocumentFragment>
  <div
    class="alert alert-success"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Everything is awesome!
    </p>
  </div>
</DocumentFragment>
`;
